README.md

```diff
@@ -177,92 +118,48 @@ ansible-galaxy collection install redhat.trusted_profile_analyzer:==0.2.0
       export TPA_EVENT_ACCESS_KEY_ID=<Kafka Username or AWS SQS Access Key>
       export TPA_EVENT_SECRET_ACCESS_KEY=<Kafka User Password or AWS SQS Secret Key>
    ```
-2. In case of Kafka Events, create environmental variable for bootstrap server
+   
+2. To choose between S3 or compatible service configure in the roles/tpa_single_node/defaults/main.yml
+3. To choose between kafka or sqs configure in the roles/tpa_single_node/defaults/main.yml
+4. To choose between Keycloak or AWS Cognito configure in the roles/tpa_single_node/defaults/main.yml
+
+5. In case of Kafka Events, create environmental variable for bootstrap server
 ```shell
 export TPA_EVENT_BOOTSTRAP_SERVER=<Kafka Bootstrap Server>
 ```
 
```

Comment:
Need to add `export TPA_STORAGE_ENDPOINT = <Minio storage URL >` for minio storage.

Reply:
Added

Reply:
@mrrajan do we have a suggestion about: 

- Ansible/Python version requirements  ? 
- CPU Ram disk space ?